skypy/utils/random.py

```diff
@@ -66,5 +66,39 @@ def _schechter_cdf(x, x_min, x_max, alpha):
     b = special.gammaincc(alpha + 1, x)
     c = special.gammaincc(alpha + 1, x_min)
     d = special.gammaincc(alpha + 1, x_max)
-
     return (a - b) / (c - d)
+
+
+def conditional_prob_shmr(x_min, x_max, size=None):
+    """ Stellar-to-halo- mass relation.
+    Sample conditional probabilities P(Mstar|Mhalo) based
+    on Birrer et al. 2018 stellar-to-halo-mass relation.
```

Comment:
You could refer to the bibliography (it will provide a link directly to the reference in the documentation). Like this:
```
""" Stellar-to-halo- mass relation.
     Sample conditional probabilities P(Mstar|Mhalo) based
     on Birrer et al. 2018 stellar-to-halo-mass relation [1]_.
```
Mind the underscore after the number.